README.md

```diff
@@ -98,13 +98,10 @@ See our [Configuration Guide](CONFIG.md) for various configuration options, incl
 Note that the validator also has a [batch processing mode](gtfs-realtime-validator-lib/README.md#batch-processing) - see the [**gtfs-realtime-validator-lib** README](gtfs-realtime-validator-lib/README.md).
  
 ## Docker
- 
-Pull the latest image from [Docker Hub](https://hub.docker.com/r/cutrusf/gtfs-realtime-validator):
-
-`docker pull cutrusf/gtfs-realtime-validator`
- 
+### Setup
+1. Download and install [Docker](https://docs.docker.com/get-started/)
+2. Pull the [latest Docker image for this project](https://github.com/orgs/MobilityData/packages/container/package/gtfs-realtime-validator). For example, `docker pull ghcr.io/mobilitydata/gtfs-realtime-validator` for the latest snapshot version of the validator.
```

Comment:
When I run:

```
docker pull ghcr.io/mobilitydata/gtfs-realtime-validator
```

... I now get:

>Error response from daemon: manifest unknown

![image](https://user-images.githubusercontent.com/928045/152564843-0849febc-880a-440a-9aad-725792e95588.png)

Reply:
As discussed today this currently fails because it's looking for the latest tag, which doesn't exist yet. So for this PR we can use `docker pull ghcr.io/mobilitydata/gtfs-realtime-validator:pr-86`, and then after this is merged we can use `docker pull ghcr.io/mobilitydata/gtfs-realtime-validator`.